aws-concepts/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: AWS Concepts | BlueFox Email
+description: Understand the AWS services used for email sending and marketing.
```

Comment:
email sending and marketing??????

I would go with something like: Learn about AWS concepts (A-Z) focusing on technologies related to email sending.